### DISCUSSION TITLE
Use of UUID for publisher_id

### DISCUSSION THREAD

**Discussion by AlwaysShreeKrishna** [oddvalue/laravel-drafts#24] [Q&A] [answered]
Since our application uses UUID for users table and when drafts columns are created with drafts() it generates publisher_id as BigInt. Is there a way I can use UUID column type for publisher_id?

**Comment by oddvalue** [comment, marked as answer]
You can manually create the columns in the migration rather than using the convenience method.
This has come up before so I might add being able to customise it to the road map.
